cpp/oneapi/dal/algo/kmeans_init/common.hpp

```diff
@@ -28,7 +28,10 @@ class descriptor_impl;
 
 namespace method {
 struct dense {};
-using by_default = dense;
+struct random_dense {};
+struct plus_plus_dense {};
+struct parallel_plus_dense {};
+using by_default = random_dense;
```

Comment:
I'm not sure if this is the ideal solution. but ok.

Reply:
The discussion continues offline.